src/Features/CSharp/Portable/CodeFixes/Suppression/CSharpSuppressionCodeFixProvider.cs

```diff
@@ -148,19 +139,20 @@ protected override SyntaxNode AddLocalSuppressMessageAttribute(SyntaxNode target
                 needsLeadingEndOfLine = true;
             }
 
-            var attributeList = CreateAttributeList(targetSymbol, diagnostic, isAssemblyAttribute: false, leadingTrivia: leadingTriviaForAttributeList, needsLeadingEndOfLine: needsLeadingEndOfLine);
+            var attributeName = suppressMessageAttribute.GenerateNameSyntax();
```

Comment:
note: i'm keeping the local suppression strategy where we don't add an import (maybe to prevent name collisions?).  We could always consider relaxing this in the future.

Reply:
I was thinking about doing that next.